server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -511,7 +517,19 @@ private DivTag renderQuestion(
         optionalToggle(
             csrfTag, programDefinitionId, blockDefinitionId, questionDefinition, isOptional);
 
+    Optional<FormTag> maybeAddressCorrectionEnabledToggle =
+        addressCorrectionEnabledToggle(
+            request,
+            csrfTag,
+            programDefinitionId,
+            blockDefinitionId,
+            questionDefinition,
+            addressCorrectionEnabled);
+
     ret.with(icon, content);
+    if (maybeAddressCorrectionEnabledToggle.isPresent()) {
+      ret.with(maybeAddressCorrectionEnabledToggle.get());
+    }
```

Comment:
suggestion (non-blocking): could simplify to 

```java
maybeAddressCorrectionEnabledToggle.ifPresent(toggle -> ret.with(toggle));
```

Reply:
Updated. Thank you for the suggestion! Also updated the existing lines below it.